quality/fca/cypress/support/api/steps/api-response-steps.ts

```diff
@@ -116,14 +116,17 @@ Then('le corps de la réponse contient un JWT', function () {
     });
 });
 
-Then('le corps de la réponse contient une erreur', function () {
-  cy.get('@apiResponse')
-    .its('body')
-    .then(async (body) => {
-      const errors = await validateDto(body, ApiErrorDto, {
-        forbidNonWhitelisted: true,
-        whitelist: true,
+Then(
+  'le corps de la réponse contient une erreur avec les champs error et error_description',
+  function () {
+    cy.get('@apiResponse')
+      .its('body')
+      .then(async (body) => {
+        const errors = await validateDto(body, ApiErrorDto, {
+          forbidNonWhitelisted: true,
+          whitelist: true,
+        });
```

Comment:
suggestion: can we avoid usage of validateDto and use the underlying functions ?

```suggestion
        const error = plainToInstance(ApiErrorDto, body);
        const validationErrors = await validate(error, {
          forbidNonWhitelisted: true,
          whitelist: true,
        });
```